src/test/java/org/opensearch/neuralsearch/processor/NeuralQueryEnricherProcessorIT.java

```diff
@@ -29,20 +29,21 @@ public class NeuralQueryEnricherProcessorIT extends BaseNeuralSearchIT {
     private static final String ingest_pipeline = "nlp-pipeline";
     private static final String TEST_KNN_VECTOR_FIELD_NAME_1 = "test-knn-vector-1";
     private final float[] testVector = createRandomVector(TEST_DIMENSION);
+    private static String modelId;
 
     @Before
```

Comment:
Shouldn't this be `@BeforeClass`?

Reply:
@BeforeClass runs once before the entire test fixture. If your test class has ten tests, @Before code will be executed ten times, but @BeforeClass will be executed only once. Here I need model Id to reinitialize after every test and a new value will be assigned.

Reply:
Then, static variable is dangerous. It shares the value across tests.

Reply:
Okay then I will remove static and just declare the variable.
private String modelId;

Reply:
Same. You need to define the variable in each and every test method itself.

Reply:
I see then I have to remove the setup method all together and and remove the teardown method as well.
I have to shift model Id code in tests and use wipeofTestResources method to clean up after every test.

Reply:
Yes.